components/uploadFormHook.tsx

```diff
@@ -37,91 +37,86 @@ export const UploadFormHook: FC = () => {
 
   return (
     <>
-      <p className="mb-3 text-2xl font-semibold">アップロードフォーム</p>
+      <h1 className="mb-3 text-2xl font-semibold">アップロードフォーム</h1>
       <form onSubmit={handleSubmit(onSubmit)}>
-        <div>
-          <div className="mb-5">
-            <p>お名前</p>
+        <div className="grid grid-cols-1 gap-6 [&>labe:input]:w-80">
+          <label className="group block">
```

Comment:
We should clarify what "*", a red star, means, so I propose this line. I used <p> tag, but I don't take care about if you change.

Reply:
I will add these lines to app/upload/page.tsx in order to clarify required.
Is it OK?
```suggestion
      <h1 className="mb-3 text-2xl font-semibold">アップロードフォーム</h1>
      <span className="before:content-['*'] before:ml-0.5 before:text-red-500">
        は必須項目です
      </span>
```

Reply:
Because ImageFormHook.tsx should have just the functions which related a image form.
Header is irrelevant a image form.
ImageFormHook.tsx's functions's are more reusable if this is small.

Reply:
OK. I got it.